ql/src/java/org/apache/hadoop/hive/ql/ddl/table/partition/drop/AbstractDropPartitionAnalyzer.java

```diff
@@ -20,25 +20,17 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.HashMap;
```

Comment:
All changes in this file are out of scope of the PR.  Please revert.

Reply:
Removed.